core-bundle/tests/Doctrine/Schema/DcaSchemaProviderTest.php

```diff
@@ -675,4 +678,46 @@ public function provideInvalidIndexDefinitions(): \Generator
             'Key definition "key path (path)" could not be parsed.',
         ];
     }
+
+    public function testAppendToSchemaIgnoresExistingMetadataDefinitions(): void
+    {
+        $dcaMetadata = [
+            'tl_page' => [
+                'TABLE_FIELDS' => [
+                    'id' => '`id` int(10) NOT NULL default 0',
+                    'title' => "`title` varchar(128) BINARY NOT NULL default ''",
+                ],
+                'SCHEMA_FIELDS' => [
+                    'published' => ['type' => 'string', 'fixed' => true, 'length' => 1],
+                ],
+                'TABLE_CREATE_DEFINITIONS' => [
+                    'published' => 'KEY `title` (`published`)',
+                ],
+            ],
+        ];
+
+        $entityMetadata = new ClassMetadata('Page');
```

Comment:
This would make phpstan happy:

```suggestion
        $entityMetadata = new ClassMetadata(get_class(new \stdClass()));
```

Reply:
Would `\Page::class` or `\stdClass::class` work too?

Reply:
Ah, yes. `\stdClass::class` works too. For some reason I didn't think this was possible :man_facepalming:.
I'll update the suggestion.